views/ng/dashboard.htm

```diff
@@ -6,12 +6,19 @@
 	        <div class="up-section">
   		        <h4 class="page-header title-color underlined">// 'summary' | tr //</h4>  
               <dl class="page-content dl-horizontal">
-        			  <dt>// 'projects' | tr //:</dt><dd>//vm.statProjects['projects']//</dd>
-        			  <dt>// 'public_projects' | tr //:</dt><dd>//vm.statProjects['public_projects']//</dd>
-        			  <dt>// 'total_projects' | tr //:</dt><dd>//vm.statProjects['total_projects']//</dd>
-        			  <dt>// 'repositories' | tr //:</dt><dd>//vm.statProjects['repositories']//</dd>
-        			  <dt>// 'public_repositories' | tr //:</dt><dd>//vm.statProjects['public_repositories']//</dd>
-        			  <dt>// 'total_repositories' | tr //:</dt><dd>//vm.statProjects['total_repositories']//</dd>
+								{{ if eq .IsAdmin 1 }}
```

Comment:
I think this section should be abstracted as a directive, and set the templateUrl as a request URL to get a specified template for showing the result. You may refer other implementations in the current project.